.vscodeignore

```diff
@@ -1,4 +1,17 @@
-.vscode/**
-.vscode-test/**
-.gitignore
-vsc-extension-quickstart.md
+# Ignore everything by default.
+*
+
+# Allow only necessary files for this package.
+!syntaxes/*.json
+!language-configuration.json
+
+# Also allow core metadata files required for packaging.
+!package.json
+!README.md
+!CHANGELOG.md
+!LICENSE
+!logo.png
+
+# Exclude common unwanted directories and files.
+.vscode/
+.github/
```

Comment:
I guess you can remove this if you ignore everything by default?

Reply:
Unfortunately `*` does not include dot files, and `.*` didn't work for me either.